mne/channels/channels.py

```diff
@@ -420,43 +420,14 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         -------
         fig : instance of matplotlib figure
             Figure containing the sensor topography.
-        selection : list
-            A list of selected channels. Only returned if ``kind=='select'``.
-
-        See Also
-        --------
-        mne.viz.plot_layout
-
-        Notes
-        -----
-        This function plots the sensor locations from the info structure using
-        matplotlib. For drawing the sensors using mayavi see
-        :func:`mne.viz.plot_trans`.
-
-        .. versionadded:: 0.12.0
-
```

Comment:
Why get rid of versionadded, notes, and see also here? It would be good to make your function smart enough to handle them (just copy?) if possible.